sources/packages/api/src-sql/Types/Add-offering-type-scholastic-standing.sql

```diff
@@ -0,0 +1,6 @@
+-- Add Scholastic Standing to sims.offering_types.
+ALTER TYPE sims.offering_types
+ADD
+    VALUE 'Scholastic Standing'
```

Comment:
Can we try to keep the default case as `Scholastic standing`. I believe that was the UI/UX recommendation.

![image](https://user-images.githubusercontent.com/61259237/165811660-8ed9a41c-f8a4-4fd5-95c9-7141d9ca7547.png)

Reply:
Okay. I believe that is the way to go forward right? @abschwenker fyi.